abletonosc/browser.py

```diff
@@ -0,0 +1,33 @@
+import os
+import sys
+import tempfile
+import Live
+import json
+from functools import partial
+from typing import Tuple, Any
+import logging
+logger = logging.getLogger("abletonosc")
+
+from .handler import AbletonOSCHandler
+
+class BrowserHandler(AbletonOSCHandler):
+    def __init__(self, manager):
+        super().__init__(manager)
+        self.class_identifier = "browser"
+        logger.info("loaded browser handler")
+
+
+    def init_api(self):
+        def get_browser_clips(self):
+            return 
+        self.osc_server.add_handler("/live/browser/test", lambda _: ("test",))
+        self.osc_server.add_handler("/live/browser/test1", partial(self._get_property, "browser", "clips"))
```

Comment:
I copied it from Song handler, but I cannot figure out where the self.song comes from. It's not in its `__init__` nor is it in `AbletonOSCHandler`'s `__init__`